Update 07-becoming-a-trusted-committer and 08-conclusion. (#262)
InnerSourceCommons/InnerSourceLearningPath: trusted-committer/07-becoming-a-trusted-committer.asciidoc

== Becoming a Trusted Committer

The Trusted Committer role is a demanding but fulfilling role. 
If this learning path interests you, you might be wondering how to actually become a Trusted Committer and if you are the right person for the job. 

InnerSource communities follow the same principles Open Source communities do, one of which is meritocracy. 
In a meritocracy, power is vested in individuals based on talent, effort, and achievements. 
This means the power and privileges that come with the Trusted Committer role need to be earned. 

> Transparency, another Open Source value, also plays a vital role in that it makes the talent, effort and achievements
> visible to the whole community.

Reviewer: ```suggestion
Transparency, another Open Source value, also plays a vital role in that it makes the talent, effort and achievements visible to the whole community.
```

The process of officially becoming a Trusted Committer differs from community to community, depends on where you are in your InnerSource journey and might evolve over time. 
In grassroots communities, the founders often assume the role of the Trusted Committer. 

> As a community grows, or in larger communities, Trusted Committers are usually nominated or voted in from the community https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[contributors].

Reviewer: Adapting the link

```suggestion
As a community grows, or in larger communities, Trusted Committers are usually nominated or voted in from the community https://innersourcecommons.org/resources/learningpath/contributor/index[contributors].
```

But the Trusted Committer role should be taken on voluntarily as it requires an immense amount of time and dedication to be successful in it.

> What are the criteria to apply in nominating https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[contributors] for a Trusted Committer role? 

Reviewer: Adapt link.

```suggestion
What are the criteria to apply in nominating https://innersourcecommons.org/resources/learningpath/contributor/index[contributors] for a Trusted Committer role? 
```

What does it take to successfully fill the role of a Trusted Committer? 
First off, potential Trusted Committers need to have demonstrated a deep technical competence during their work in the community. 
In addition to that, they must have proven their ability to effectively communicate with peers in the community and ideally also with
product owners and with management.

In the same vein, they must have shown the willingness and patience to use their skills and spend intentional time upleveling contributors.

> Finally, fulfilling the Trusted Committer role requires a certain emotional maturity to deal with stressful social situations, which are
> bound to come up from time to time. 

Reviewer: ```suggestion
Finally, fulfilling the Trusted Committer role requires a certain emotional maturity to deal with stressful social situations, which are bound to come up from time to time. 
```

Contributors who satisfy these criteria will be good potential Trusted Committers, in our opinion.

The Trusted Committer role might not appear all that attractive to some contributors as it means spending less time coding. 
Being nominated for the Trusted Committer role might even be perceived by some as a demotion or as negative feedback on their coding skills. 
But the opposite is true. 
Being nominated for the Trusted Committer role usually means someone has recognized your valuable contributions and sees in you the potential to grow and lead. 

> The Trusted Committer role will give you more influence over the evolution of the codebase and will ultimately make you a more complete
> developer. 

Reviewer: ```suggestion
The Trusted Committer role will give you more influence over the evolution of the codebase and will ultimately make you a more complete developer. 
```

Explaining to contributors how the software works more often than not leads to new insights on part of the Trusted Committer and will help to identify opportunities to improve the software.

Whether you have one or multiple Trusted Committers depends on the size and the risk associated with the software developed by the InnerSource community. 
The Trusted Committer role is time consuming, and not everyone is willing or able to make that type of commitment. 
Because of this, some companies use a _Trusted Committer rotation_ system where multiple Trusted Committers share the workload of the Trusted Committer role, and the Trusted Committers who are not _on duty_ can exclusively focus on tech-oriented work. 

> Having more than one Trusted Committer also makes it easier when someone leaves the company or moves on from the role to do something else.

Reviewer: I actually liked having the "In that case, it is important	
that there are other Trusted Committers in place already, who can take over and ensure	
continuity in the community." sentence as additional part here.

Author: added back

Because of this, some companies use a _Trusted Committer rotation_ system where multiple Trusted Committers share the workload of the Trusted Committer role, and the Trusted Committers who are not _on duty_ can exclusively focus on tech-oriented work. 
Having more than one Trusted Committer also makes it easier when someone leaves the company or moves on from the role to do something else.

In summary, the Trusted Committer role has to be earned by making valuable contributions - both technical and social - for the benefit of the community. 
In a healthy community, you will have fellow Trusted Committers at your side. 
As a Trusted Committer, you will have less time to code, but by acting as a force multiplier you will ultimately be able to boost your value contribution to the community and accelerate your own growth.

The process of officially becoming a Trusted Committer differs from community to community,
depends on where you are in your InnerSource journey and might evolve over
time. In grass-roots type communities, the founders often automatically assume
the role of the Trusted Committer as well. As a community grows, the community or the

> existing Trusted Committers might nominate a https://innersourcecommons.org/resources/learningpath/contributor/index[_contributor_] to become Trusted Committer, which might or might

Reviewer 2: Keep all of the links pointing at `innersourcecommons.org`.

Reviewer: Fixed this by means of commit suggestions, hence resolving this discussion.
